lib/plugins/helper/toc.js

```diff
@@ -17,14 +17,15 @@ function tocHelper(str, options = {}) {
   const className = options.class;
   const listNumber = options.list_number;
 
-  let result = `<ol class="${className}">`;
+  let result = htmlTag('ol', { class: className });
   const lastNumber = [0, 0, 0, 0, 0, 0];
   let firstLevel = 0;
   let lastLevel = 0;
 
   for (let i = 0, len = data.length; i < len; i++) {
     const el = data[i];
     const { level, id, text } = el;
+    const href = id ? `#${id}` : id;
```

Comment:
alternative approach is to 

```
id ? `#${id}` : '';
```

to output `<a href="">` to make it appear as a link. without `href`, `<a>` simply appear as normal text.